setup.py

```diff
@@ -45,6 +40,7 @@
       packages=['pynvim', 'pynvim.api', 'pynvim.msgpack_rpc',
                 'pynvim.msgpack_rpc.event_loop', 'pynvim.plugin',
                 'neovim', 'neovim.api'],
+      python_requires=">=3.6",
```

Comment:
This should be `>=3.7` as well

Reply:
As mentioned in the comment above, this is because bumping the `python_requires` breaks the CI.